core/agent.py

```
@@ -30,6 +30,63 @@ def __init__(
         self.conversation: List[Dict[str, str]] = []
         self._stop_condition: Optional[Callable[[], bool]] = None
 
+    def _should_use_chain(self, task: str) -> bool:
+        """判断是否应该使用提示链模式"""
+        # 优先级1：明确提到"提示链"
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)